device/jsonld-contexts/device-compound.jsonld

```diff
@@ -0,0 +1,7 @@
+{
+  "@context": [
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/device/jsonld-contexts/edgespot.jsonld",
+    "https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/device/jsonld-contexts/device.jsonld",
```

Comment:
start using the ones on https://easy-global-market.github.io/ngsild-api-data-models/ (e.g., https://easy-global-market.github.io/ngsild-api-data-models/device/jsonld-contexts/edgespot.jsonld)